aurora/grids_utils.py

```diff
@@ -747,25 +751,42 @@ def build_aug_geqdsk(shot, time):
     geqdsk['fluxSurfaces']['Z0'] = Z0
     geo = geqdsk['fluxSurfaces']['geo'] = {}
     
-    pf = equ.pfl[it,:]
+    pf = eqm.pfl[it,:]
     
-    geo['psin'] = (pf - equ.psi0[it]) / (equ.psix[it] - equ.psi0[it])
+    geo['psin'] = (pf - eqm.psi0[it]) / (eqm.psix[it] - eqm.psi0[it])
     ind = geo['psin'] <= 1
     geo['psin'] = geo['psin'][ind]
-    geo['surfArea'] = equ.area[it, ind]
-    geo['vol'] = equ.vol[it, ind]
-    geo['rhon'] = equ.rho_tor_n[it,ind]
+    geo['surfArea'] = eqm.area[it, ind]
+    geo['vol'] = eqm.vol[it, ind]
+    geo['rhon'] = eqm.rho_tor_n[it,ind]
     
-    qpsi = abs(equ.q)[it]
+    qpsi = abs(eqm.q)[it]
     qpsi[-1] = 2 * qpsi[-2] - qpsi[-3]
     geqdsk['QPSI'] = qpsi[ind]
     
     mdpl = geqdsk['fluxSurfaces']['midplane'] = {}
-    mdpl['R'] = sf.rhoTheta2rz(equ, pf[ind], 0, t_in=time, coord_in='Psi')[0][0, 0]
+    mdpl['R'] = sf.rhoTheta2rz(eqm, pf[ind], 0, t_in=time, coord_in='Psi')[0][0, 0]
     
     # separatrix
-    r, z = sf.rho2rz(equ, t_in=time, rho_in=0.999, all_lines=False)
+    r, z = sf.rho2rz(eqm, t_in=time, rho_in=0.999, all_lines=False)
     
     geqdsk['RBBBS'], geqdsk['ZBBBS'] = r[0][0], z[0][0]
+    
+    # a few extra things to enable greater use of omfit_eqdsk
+    geqdsk.add_rhovn()
+
+    geqdsk['PRES'] = eqm.pres[it,ind]
+    geqdsk['PPRIME'] = eqm.dpres[it,ind]  # tentative
+    geqdsk['FFPRIM'] = eqm.ffp[it,ind]
+    #embed()
```

Comment:
Make sure your gEQDSK is in COCOS 1, and add `AuxQuantities` and `FluxSurfaces`
```python
geqdsk._cocos=COCOS_OF_THE_QUANTITIES_THAT_YOU_ENTERED
geqdsk.cocosify(1, calcAuxQuantities=True, calcFluxSurfaces=True)
```

Reply:
I had to play around quite a bit to get this to work -- I'm almost certain that I made up some quantities... but now `geqdsk.cocosify` runs and that's at least some progress.